packages/fxa-react/lib/test-utils/test.ftl

```diff
@@ -0,0 +1,11 @@
+-term = Mozilla
```

Comment:
I know this doesn't cover every single use case (like [selectors](https://projectfluent.org/fluent/guide/selectors.html)) we've got but this does cover almost all of them, and I think all of them for Settings. I figure we can add tests and tweak the test util if needed when we get there.